homeassistant/components/august/activity.py

```diff
@@ -140,11 +145,25 @@ def async_schedule_house_id_refresh(self, house_id: str) -> None:
 
         debouncer = self._update_debounce[house_id]
         debouncer.async_schedule_call()
+
         # Schedule two updates past the debounce time
         # to ensure we catch the case where the activity
         # api does not update right away and we need to poll
         # it again. Sometimes the lock operator or a doorbell
         # will not show up in the activity stream right away.
+        # Only do additional polls if we are past
+        # the initial lock resync time so avoid a storm
```

Comment:
maybe "to" instead of "so"?
```suggestion
        # the initial lock resync time to avoid a storm
```